### DISCUSSION TITLE
Elenco di tutti gli episodi

### DISCUSSION THREAD

**Discussion by agofa** [pdonadeo/e-uscito-joypad#52] [Ideas]
Attualmente sono elencati gli ultimi 8 episodi
Se si fa una ricerca si vede il risultato
Se la ricerca è molto generica (per esempio "the "), l'elenco comprende molti episodi (anzi direi tutti)

Sarebbe utile un pulsante, un coso, un oggetto, un boh? che mostri elenco con tutti gli episodi, ordinati dal più recente

Chissà se mi sono capito?
@pdonadeo e @MarcoJul

**Comment by pdonadeo** [comment]
C'è già aperta una issue su questo punto, la #37.

Se leggi lì si era detto di risolvere con la paginazione perché comunque sia caricare tutti gli episodi è impossibile.

Io aspetterei di vedere come viene il discorso della ricerca con autocompletamento del titolo dei giochi per procedere sul resto.

**Reply by agofa** [reply]
Hai ragione butta via questa
